<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingForm</class>
 <widget class="QDialog" name="SettingForm">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>307</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>上傳設定</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="font">
      <font>
       <family>Microsoft JhengHei UI</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="title">
      <string>伺服器設定</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>Microsoft JhengHei UI</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>伺服器</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_server">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_editServerURL">
          <property name="font">
           <font>
            <family>Microsoft JhengHei UI</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>修改</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>Microsoft JhengHei UI</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>密碼</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_pwd">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_editPwd">
          <property name="font">
           <font>
            <family>Microsoft JhengHei UI</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>修改</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>Microsoft JhengHei UI</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>參數</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_param">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_editParam">
          <property name="font">
           <font>
            <family>Microsoft JhengHei UI</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>修改</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <family>Microsoft JhengHei UI</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Timeout 設定</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <family>Microsoft JhengHei UI</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>上傳timeout</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_timeout">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>9</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_editTimeout">
          <property name="font">
           <font>
            <family>Microsoft JhengHei UI</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>修改</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="font">
           <font>
            <family>Microsoft JhengHei UI</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer">
       <property name="font">
        <font>
         <family>Microsoft JhengHei UI</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_closeSet">
       <property name="font">
        <font>
         <family>Microsoft JhengHei UI</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>關閉</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
